azerothcore/azerothcore-wotlk
# fix(Core/MiscHandler): Dungeon requirements Message

## Changes Proposed:
-  Dungeon requirements Level and Item, SendAreaTriggerMessage variable %s, %u  error

## Issues Addressed:

- fix dungeon access requirements message error
- fix level variable always 0 and item name get locale not work

## SOURCE:


- Original:
![1](https://github.com/azerothcore/azerothcore-wotlk/assets/134824156/8f182c73-e326-429d-bbbb-0119003830ab)

- Fixed:
![2](https://github.com/azerothcore/azerothcore-wotlk/assets/134824156/2a754b9d-bb0b-41cd-939d-e77c5a810b34)

## Tests Performed:


- It builds without errors
- Tested in-game

- Server:
AzerothCore rev. https://github.com/azerothcore/azerothcore-wotlk/commit/893a9beeacc4554f9ac85755b40520a9b8e6d411+
Debian 11 x64
MySQL 8.0.33

- Client:
3.3.5a.12340 zhCN
Windows 10 x64 LTSC

## How to Test the Changes:


1. SELECT * FROM acore_string WHERE entry = 50;
2. check "content_default" AND "locale_zhCN" value
3. in-game, and .go xyz  -301.8008 3162.2664 31.720385 530
4. Try to enter the instance, observe message.
